### PR TITLE
cEP 6: coala Community Code of Conduct

### PR DESCRIPTION
**Pull request by Mariatta**
coala Community Code of Conduct

Closes https://github.com/coala/cEPs/issues/58

### CODE REVIEW

File: cEP-0006.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+of us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces managed by the coala project. This
+includes gitter, github, gitlab, the issue tracker, coala events, and any other
+forums created by the project team which the community uses for communication.
+In addition, violations of this code outside these spaces may affect a person's
+ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you
+[report it](community@coala.io).
```
Comment (sils) [inline cEP-0006.md]: I think you wanna have a mailto: there

Comment (sils) [reply cEP-0006.md]: hm now I'm a bit afraid of spam, maybe just `community AT coala DOT io` as text or so?

Comment (Mariatta) [reply cEP-0006.md]: Done. Thanks :)

**Comment by Adrianzatreanu** [comment]
holy, i love this (maybe less the part with no racist jokes)
ack 764a1321988b1960272dad66a7dc72b71a0434c6

**Comment by Adrianzatreanu** [comment]
ack 764a1321988b1960272dad66a7dc72b71a0434c6

**Comment by Adrianzatreanu** [comment]
needs 1 more ack afaik tho

File: cEP-0006.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+coala Community Code of Conduct
+===============================
+
+|Metadata|                                   |
+|--------|-----------------------------------|
+|cEP     |0006                               |
+|Version |1.1                                |
```
Comment (sims1253) [inline cEP-0006.md]: wouldn't this be 1.0 after merge?

File: cEP-0006.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+|Title   |coala Community Code of Cpnduct    |
```
Comment (sims1253) [inline cEP-0006.md]: Cpnduct typo

File: cEP-0006.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+issues and unhappiness. To that end, we have a few ground rules that we ask
+people to adhere to. This code applies equally to founders, mentors,
+contributors, and those seeking help and guidance.
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in
+the spirit in which it’s intended - a guide to make it easier to enrich all
+of us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces managed by the coala project. This
+includes gitter, github, gitlab, the issue tracker, coala events, and any other
```
Comment (sims1253) [inline cEP-0006.md]: Gitter, GitHub, Gitlab

File: cEP-0006.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+forums created by the project team which the community uses for communication.
```
Comment (sims1253) [inline cEP-0006.md]: what is the project team? the project's team?

File: cEP-0006.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+report it by emailing `community AT
+coala DOT io`.
+
+* **Be friendly and patient.**
+* **Be welcoming.** We strive to be a community that welcomes and supports
+  people of all backgrounds and identities. This includes, but is not limited
+  to members of any race, ethnicity, culture, national origin, colour,
+  immigration status, social and economic class, educational level, sex,
+  sexual orientation, gender identity and expression, age, size, family status,
+  political belief, religion, and mental and physical ability.
```
Comment (sims1253) [inline cEP-0006.md:46]: maybe add editor choice to the list? :)

Comment (Mariatta) [reply cEP-0006.md:46]: nope

File: cEP-0006.md (hunk 5 above)
Comment (Mariatta) [reply cEP-0006.md]: maybe it should be
`created by the coala team`

File: cEP-0006.md (hunk 3 above)
Comment (Mariatta) [reply cEP-0006.md]: oops

File: cEP-0006.md (hunk 2 above)
Comment (Mariatta) [reply cEP-0006.md]: will change to 1.0

**Comment by sims1253** [comment]
ack a7a27ad

**Comment by sims1253** [comment]
technically this needs a second ack

**Comment by Adrianzatreanu** [comment]
ack a7a27ad

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**